### PR TITLE
fix the slo-reporter configmap and cronjob

### PR DESCRIPTION
**Pull request by harshad16**
fix the slo-reporter configmap and cronjob
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

### CODE REVIEW

File: slo-reporter/overlays/zero-prod-ocp4-stage/configmap.yaml [hunk 1]
```diff
@@ -16,12 +22,25 @@ data:
...
+apiVersion: v1
+metadata:
+  name: ceph
+data:
   bucket-name: "thoth"
   bucket-prefix: "data"
   public-bucket-name: DH-PLAYPEN
   host: "https://s3-openshift-storage.apps.zero.massopen.cloud/"
   endpoint: "https://s3-openshift-storage.apps.zero.massopen.cloud/"
   host-url: "http://prometheus-portal-thoth-infra-prod.apps.zero.massopen.cloud"
```
Comment (pacospace) [inline slo-reporter/overlays/zero-prod-ocp4-stage/configmap.yaml:37]: this should go to prometheus CM

Comment (harshad16) [reply slo-reporter/overlays/zero-prod-ocp4-stage/configmap.yaml:37]: which one @pacospace ?

Comment (pacospace) [reply slo-reporter/overlays/zero-prod-ocp4-stage/configmap.yaml:37]: `host-url: "http://prometheus-portal-thoth-infra-prod.apps.zero.massopen.cloud"`

Comment (harshad16) [reply slo-reporter/overlays/zero-prod-ocp4-stage/configmap.yaml:37]: cool, we can open that in a different pr, that particular is not used in cronjob so it should fail it

**Pull request merged**